server/src/service/BoardService.js

```diff
@@ -10,4 +11,32 @@ export class BoardService extends BaseService {
 
         return BoardService.instance;
     }
+
+    async getMyBoards(id) {
+        const boards = await this.boardRepository.find({
+            select: ['id', 'title'],
+            where: { creator: id },
+        });
+        return boards;
+    }
+
+    async getInvitedBoards(id) {
+        const boards = await this.boardRepository
+            .createQueryBuilder('board')
+            .select('board.id')
+            .addSelect('board.title')
+            .innerJoin('board.invitations', 'invitation', 'invitation.user=:userId', { userId: id })
+            .getMany();
+        return boards;
+    }
+
+    @Transactional()
+    async getBoardsByUserId(userId) {
+        const data = {};
+
+        data.myBoards = await this.getMyBoards(userId);
+        data.invitedBoards = await this.getInvitedBoards(userId);
```

Comment:
```suggestion
        const promises = [this.getMyBoards(userId), this.getInvitedBoards(userId)];
        const [myBoards, invitedBoards] = await Promise.all(promises);
        return { myBoards, invitedBoards };
```
nodejs의 비동기 IO를 최대한 활용해보는건 어떨까요? 
37번 라인과 38번 라인은 동기적으로 (순차적으로) 실행되지 않아도 되는 로직이라서 동시에 요청을 보내면 효율적일 것 같다고 생각합니다.

Reply:
아직은 비동기가 친숙하지 않아서 그런 생각은 하지 못했네요.
좋은 의견 감사합니다!:+1: